app/controllers/idv/capture_doc_controller.rb

```diff
@@ -22,7 +22,8 @@ def ensure_user_id_in_session
     end
 
     def add_unsafe_eval_to_capture_steps
-      return unless %w[mobile_front_image capture_mobile_back_image].include?(params[:step])
+      step = params[:step]&.underscore
```

Comment:
yikes my bad, I must have missed this access when I refactored for the URL. I made a `current_step` helper method that does this (in `FlowStateMachine`) can we just do that here?

```ruby
return unless %w[mobile_front_image capture_mobile_back_image].include?(current_step)
```

Reply:
I think we can